arb_os/evmOps.mini

```diff
@@ -272,7 +273,7 @@ public impure func evmOp_extcodehash(addrAsUint: uint) -> bytes32 {
     let addr = address(addrAsUint);     // truncates if necessary
     if let Some(topFrame) = evmCallStack_topFrame() {
         let acct = evmCallFrame_getAccount(topFrame, addr);
-        if (account_isEmpty(acct)) {
+        if (! account_hasContract(acct)) {
```

Comment:
I think the old behavior might actually have been correct here, but I'm not sure

Reply:
Yeah, it looks like what is left of this PR (after merging in changes from develop) is incorrect.

Reply:
Yes, will fix.